adapters/Vungle/VungleAdapter/GADMAdapterVungleInterstitial.m

```diff
@@ -180,4 +180,8 @@ - (void)willLeaveApplication {
   [_connector adapterWillLeaveApplication:self];
 }
 
+- (void)rewardUser {
+  NSLog(@"%@ received rewardUser event.", NSStringFromClass(self.class));
```

Comment:
Same thing for these other events that do nothing.

Reply:
@ericleich Updated, thanks a lot for the guidance. ^_^